website/content/docs/platform/k8s/helm/run.mdx

```diff
@@ -436,7 +436,10 @@ running:
 $ kubectl delete pod <name of Vault pod>
 ```
 
-If Vault is deployed using `ha` mode, the standby pods must be upgraded first.
+If Vault is deployed using `ha` mode, the standby pods must be upgraded first. Before deleting the pod, make sure to remove the node from the raft by running `vault operator raft remove-peer <server_id>`. To confirm that the node has been successfully removed from Raft, you can run vault operator raft list-peers. Only after confirming the removal should you delete the pod.
+
+> If you remove the pod directly without properly removing the node from the cluster, Raft will be unaware of the correct number of nodes in the cluster. This can trigger a leader election, potentially causing downtime.
```

Comment:
```suggestion

If you deployed Vault in high availability (`ha`) mode, you must upgrade your
standby pods before upgrading the active pod. 

Before deleting the standby pod, remove the associated node from the raft with
`vault operator raft remove-peer <server_id>`.

To confirm that the node was successfully removed from Raft, run
`vault operator raft list-peers`.

Once you confirm the removal succeeded, you can delete the pod.

<Warning title="Delete nodes to avoid unnecessary leader elections">

Removing a pod without deleting the node from its cluster first means that the
number of nodes Raft
will be unaware of the correct number of nodes in the cluster. This can trigger a leader election, potentially causing downtime.

</Warning>
```

Style correction: write in active voice, make explicit instructions easy to see at a glance.